### PR TITLE
[VSPC-273] added stylesheet to show the remove icon

### PR DESCRIPTION
**Pull request by pooja-thalur**
# Guidelines for Pull Requests

If you haven't yet read our code review guidelines, please do so, You can find them [here](https://diging.atlassian.net/wiki/spaces/DIGING/pages/2256076801/Code+Review+Guidelines).

Please confirm the following by adding an x for each item (turn `[ ]` into `[x]`).

- [x] I have removed all code style changes that are not necessary (e.g. changing blanks across the whole file that don’t need to be changed, adding empty lines in parts other than your own code)
- [x] I am not making any changes to files that don’t have any effect (e.g. imports added that don’t need to be added)
- [x] I do not have any sysout statements in my code or commented out code that isn’t needed anymore
- [x] I am not reformatting any files in the wrong format or without cause. 
- [x] I am not changing file encoding or line endings to something else than UTF-8, LF
- [x] My pull request does not show an insane amount of files being changed although my ticket only requires a few files being changed
- [x] I have added Javadoc/documentation where appropriate
- [x] I have added test cases where appropriate
- [x] I have explained any part of my code/implementation decisions that is not be self-explanatory

## Please provide a brief description of your ticket 
(you can copy the ticket if it hasn't changed) 
[Image tags cannot be remove anymore](https://diging.atlassian.net/browse/VSPC-273)

...Put description here...
Somewhere along the way this seems to have been broken. The icon still seems to be there but it’s not shown.
  
## Anything else the reviewer needs to know?

... describe here ...

### CODE REVIEW

**Comment by jdamerow** [comment]
Make it so, Jenkins.

File: vspace/src/main/webapp/WEB-INF/views/staff/images/imagelist.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <html layout:decorate="~{layouts/main_staff}">
 <head>
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/images/imagelist.html:3]: Let's remove it from here and put it in the layout template and then also remove it from file.html and slide.html.

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: vspace/src/main/webapp/WEB-INF/views/staff/files/file.html [hunk 2]
```diff
@@ -3,8 +3,6 @@
 <link rel="stylesheet" type="text/css" id="applicationStylesheet"
     th:href="@{/resources/extra/Home.css}" />
 
-<script src="//use.fontawesome.com/releases/v5.8.1/js/all.js"
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/files/file.html:6]: is the js needed somewhere? if so, it needs to be added to the layout as well.

**Review by jdamerow — CHANGES_REQUESTED** [review]

File: vspace/src/main/webapp/WEB-INF/views/staff/files/file.html (hunk 2 above)
Comment (pooja-thalur) [reply vspace/src/main/webapp/WEB-INF/views/staff/files/file.html:6]: added it in both the layout files - main.html and main_staff.html

File: vspace/src/main/webapp/WEB-INF/views/layouts/main.html [hunk 3]
```diff
@@ -25,16 +25,18 @@
...
 <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
 <!--[if lt IE 9]><script src="<c:url value="/resources/bootstrap/assets/js/ie8-responsive-file-warning.js" />"></script><![endif]-->
 <script th:src="@{/resources/bootstrap/assets/js/ie-emulation-modes-warning.js}"></script>
-
 <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
 <!--[if lt IE 9]>
       <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+<script src="//use.fontawesome.com/releases/v5.8.1/js/all.js" data-auto-replace-svg="nest"></script>
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/layouts/main.html:38]: should be the same version as css

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Comment by jdamerow** [comment]
Image tags can't be deleted. There is an error in the JS console.

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Pull request merged**